### PR TITLE
update google-cloud-bigquery dependency range

### PR DESCRIPTION
**Pull request by tswast**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

Towards ibis-project/ibis#2413

### CODE REVIEW

**Comment by tswast** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -26,15 +26,14 @@ requirements:
     - clickhouse-sqlalchemy
     - geoalchemy2
     - geopandas
-    # see https://github.com/googleapis/python-api-core/issues/47
-    - google-api-core =1.17.0
-    - google-cloud-bigquery >=1.0.0
+    - google-cloud-bigquery >=1.12.0,<2.0.0dev
```
Comment (jreback) [inline recipe/meta.yaml]: perfect!

**Comment by jreback** [comment]
thanks @tswast ping me when this is green (or at least *some* of the builds work lol)

**Comment by tswast** [comment]
@jreback All green!

Might need to revert 09af52b9b52b89fb5d87b7260f5a8720ec4dce70 though based on my understanding of https://github.com/conda-forge/ibis-framework-feedstock/pull/47

**Comment by jreback** [comment]
thanks @tswast

**Pull request merged**